app.test.js

```diff
@@ -96,6 +96,26 @@ describe('getReviewsById', () => {
           category: expect.any(String),
           created_at: expect.any(String),
           votes: expect.any(Number),
+          comment_count: 0
+        });
+      });
+  });
+  test("Responds with a review object with correct properties for comments", () => {
+    return request(app)
+      .get("/api/reviews/2")
+      .expect(200)
+      .then(({body}) => {
+        expect(body.review).toMatchObject({
+          review_id: 2,
+          title: expect.any(String),
+          designer: expect.any(String),
+          owner: expect.any(String),
+          review_img_url: expect.any(String),
+          review_body: expect.any(String),
+          category: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          comment_count: expect.any(Number)
```

Comment:
You can use an exact value for comment_count here, or even the whole object. Because you're using the review_id of 2, you know exactly what review you're getting, so this test can afford to be more brittle.